src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -1486,21 +1490,25 @@ public PropertyDescriptor getColumn(Domain domain, String columnName)
     public Map<String, Object> getPropertyValidator(PropertyDescriptor column, String name)
     {
         List<Map<String, Object>> validators = (ArrayList<Map<String, Object>>)column.getAllProperties().get("propertyValidators");
-        Map<String, Object> validator = validators.stream()
+        List<Map<String, Object>> filteredList = validators.stream()
                 .filter(a-> a.get("name").equals(name))
-                .collect(Collectors.toList())
-                .get(0);
-        return validator;
+                .collect(Collectors.toList());
+        if (filteredList.size() == 0)
+            return null;
+        else
+            return filteredList.get(0);
```

Comment:
I don't think this will fix the failure on TeamCity. It will just throw an NPE instead of array index exception.

Reply:
So, should we assume that null is an error condition and provide an error?
Okay, doing that.